proteus/BoundaryConditions.pyx

```diff
@@ -47,7 +47,7 @@ class BC_Base(object):
             cls.ct = context
 
 
-class BoundaryCondition():
+class BoundaryCondition(BC_Base):
```

Comment:
@tridelat I accidentally committed this change while trying to debug the testing. It looked strange to me that you had this BoundaryConditions class but it wasn't derived from BC_Base. Can you check  this to make sure I switch it to what you intended?